django_auth_ldap/backend.py

```diff
@@ -78,6 +78,27 @@
 ldap_error = django.dispatch.Signal()
 
 
+_error_context_descriptions = {
+    'get_group_permissions': 'loading group permissions',
+    'search_for_user_dn': 'looking up user',
+    'mirror_groups': 'updating mirrored groups',
+}
+
+
+def _report_error(sender, context, user, request, exception):
+    results = ldap_error.send(
+        sender,
+        context=context,
+        user=user,
+        request=request,
+        exception=exception,
+    )
+    if len(results) == 0:
+        description = _error_context_descriptions.get(context, "from unknown context")
+        logger.warning("Caught LDAPError %s: %s", description, pprint.pformat(exception))
```

Comment:
This logic of "only log warnings when no signal handlers were present" is taken from `get_group_permissions()`. Maybe it shouldn't be conditional, but warnings should be logged always?

Reply:
I think the warnings should be logged anyway.
Informing developers of errors is important, and Python offers multiple built-in ways to configure loggers.
We might want to use a sub logger in order to help dealing with the warnings, but that does not seem required for this PR.